src/main/java/com/microsoft/sqlserver/jdbc/StreamColumns.java

```diff
@@ -311,21 +318,26 @@ private SensitivityClassification processDataClassification(TDSReader tdsReader)
                     informationType = informationTypes.get(informationTypeIndex);
                 }
 
-                if (TDS.MAX_SUPPORTED_DATA_CLASSIFICATION_VERSION <= tdsReader
-                        .getServerSupportedDataClassificationVersion()) {
-                    sensitivityRank = tdsReader.readInt();
-                    if (!SensitivityRank.isValid(sensitivityRank)) {
+                int sensitivityRankProperty = SensitivityRank.NOT_DEFINED.getValue();
+                if (sensitivityRankSupported) {
+                    sensitivityRankProperty = tdsReader.readInt();
+                    if (!SensitivityRank.isValid(sensitivityRankProperty)) {
                         tdsReader.throwInvalidTDS();
                     }
+                    // add sensitivity properties for the source
```

Comment:
Just to confirm - this line (328) was executed regardless of sensitivity rank being supported, but now it's only being executed if sensitivity rank is supported because it moved up one if statement bracket. Is this intended?